src/imitation/algorithms/adversarial.py

```diff
@@ -147,65 +160,104 @@ def gen_policy(self) -> BaseRLModel:
     """Policy (i.e. the generator) being trained."""
     return self._gen_policy
 
-  def train_disc(self, n_steps=10, **kwargs):
+  def train_disc(self, total_timesteps: Optional[int] = None) -> None:
```

Comment:
I think `batch_size` might be a clearer name for this than `total_timesteps`. When I hear "total" I think complete number of timesteps, across all training.

Reply:
`n_samples` is another option, since we're not really taking new timesteps in the environment.

Reply:
fair point, this is conceptually different from what `total_timesteps` usually means. `n_samples` sounds good to me.